processor/README.md

```diff
@@ -187,16 +201,147 @@ These parameters are already provided by the cart created in composable commerce
+Submits details for a payment to Adyen platform to confirm a payment. It is only necessary when the payment is initialized through [create payment](#create-payment)
+#### Endpoint
+`POST /payments/details`
+
+#### Request Parameters
+The request body is same as [adyen checkout create payment details request](The request body is same as [adyen checkout create payment request](https://docs.adyen.com/api-explorer/Checkout/71/post/payments#request)) with following parameters :
+- authenticationData: Data for 3DS authentication.
+- details: A collection of result returned from the `/payments` call.
+- paymentData: Encoded payment data returned from the `/payments` call. If `AuthenticationNotRequired` is received as `resultCode` in the `/payments` response, use the `threeDSPaymentData` from the same response. If the `resultCode` is `AuthenticationFinished`, use the `action.paymentData` from the same response.
+
+#### Response Parameters
+It returns following attributes in response
+- 
+- 
+- paymentReference: Unique identifier of payment resources updated in commercetools composable commerce.
+-  
 
 ### Get supported payment components
-TBC
+Private endpoint protected by JSON Web Token that exposes the payment methods supported by the connector so that checkout application can retrieve the available payment components. 
+#### Endpoint
+`GET /operations/payment-components`
 
-### Get status
-TBC
+#### Request Parameters
+N/A
+
+#### Response Parameters
+Now the connector supports payment methods such as `card`, `iDEAL`, `PayPal` and `Sofort`
+```
+{
+    components: [
+        {
+          type: 'card',
+        },
+        {
+          type: 'ideal',
+        },
+        {
+          type: 'paypal',
+        },
+        {
+          type: 'sofort',
+        },
+    ],
+}
+```
 
 ### Get config
-TBC
+Exposes configuration to the frontend such as `clientKey` and `environment`.
+#### Endpoint
+`GET /operations/config`
+
+#### Request Parameters
+N/A
 
+#### Response Parameters
+It returns an object with `clientKey` and `environment` as key-value pair as below:
+```
+{
+  clientKey: <clientKey>,
+  environment: <environment>,
+}
+```
+
+
+### Get status
+It provides health check feature for checkout front-end so that the correctness of configurations can be verified.
+#### Endpoint
+`GET /operations/status`
+
+#### Request Parameters
+N/A
+
+#### Response Parameters
+It returns following attributes in response:
+- status: It indicates the health check status. It can be `OK`, `Partially Available` or `Unavailable`
+- timestamp: The timestamp of the status request
+- version: Current version of the payment connector.
+- checks: List of health check result details. It contains health check result with various external system including commercetools composable commerce and Adyen payment services provider.
+```
+    [ 
+        {
+            name: <name of external system>
+            status: <status with indicator UP or DOWN>
+            details: TBC
```

Comment:
@dasanorct need you help to input here. I am not sure what would be included inside this attribute.

Reply:
here, if it's UP then we are returning the list of payment methods enabled in adyen but this is just a sample request to adyen in order to check that the connection works and it is down, we return the error. 
maybe we should change the code to not return the payment methods when it's UP as it's irrelevant in my opinion and keep the details only for the DOWN status to provide info about the error.

what do you think, @ddeliziact ?

Reply:
@leungkinghin-ct @dasanorct this field is an open field to add additional information either is UP or DOWN. It will depend on the status check. If we want to make it more strict and only have that field in case of error we can do that and making it a string we can do that (will not affect checkout as we are not using it).

Reply:
@dasanorct @ddeliziact For the case of documentation, I just feel curious what properties would be included in the `details` JSON object, so the doc reader would know what kind of information can be obtained inside `details`.

Let me summerize it as `additional information for connection checking`. wdyt?

For technical modification, i believe we can leave it for future.

Reply:
@dasanorct @ddeliziact 
https://github.com/commercetools/connect-payment-integration-adyen/pull/104/commits/fdfda8098b2ce4cc8bcf4d17ff18bf2f0db8b07d